strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -712,11 +723,63 @@ public void testLastVersionManagement(TestInfo testInfo)
         artifactEntryWithClassifier = artifactEntryService.findOneArtifact(STORAGE0,
                                                                            repositoryId,
                                                                            artifactPathWithClassifier);
-                                                                           
+
         MatcherAssert.assertThat(artifactEntryWithClassifier.getTagSet(), CoreMatchers.notNullValue());
         MatcherAssert.assertThat(artifactEntryWithClassifier.getTagSet().size(), CoreMatchers.equalTo(0));
     }
 
+    @Test
+    public void testChecksumsStorage(TestInfo testInfo)
+            throws Exception
+    {
+        String repositoryId = getRepositoryName("checksums-storage", testInfo);
+
+        MutableRepository repository = mavenRepositoryFactory.createRepository(repositoryId);
+        repository.setLayout(Maven2LayoutProvider.ALIAS);
+
+        createRepositoryWithArtifacts(STORAGE0,
```

Comment:
in your case you should use `createRepository` instead of `createRepositoryWithArtifacts` because the artifact will be deployed within test.